api/schemas/webhook.json

```diff
@@ -86,6 +86,15 @@
         "verbose_storage": {
             "description": "Whether to include storage metadata in the `get_urls` property in `flows/segments_added` events. This option is the same as the `verbose_storage` query parameter for the /flows/{flowId}/segments API endpoint.",
             "type": "boolean"
+        },
+        "status": {
```

Comment:
Is it intentional that this is setable by the client? I can see use cases for that.

Reply:
I think what you've done is enough. Just wanted to double check as it went beyond reporting errors :)